data/strings/strings.txt

```diff
@@ -23342,54 +23248,54 @@
     zh-Hant = 載入地圖
 
   [pref_tts_speedcams_auto]
-    comment = Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit
-    tags = android
-    en = Warn if there is a risk of exceeding the speed limit
-    af = Waarsku indien daar ’n risiko is dat die grens oorskry word
-    ar = حذرني  إذا هناك احتمالية تجاوز الحد الأقصى للسرعة
-    az = Sürət həddini aşmaq riski varsa, xəbərdarlıq edin
-    be = Папярэджваць калі ёсць рызыка перавышэння дазволенай хуткасці
-    bg = Предупреждава ако има риск от превишаване на ограничението на скоростта
-    ca = Avisa si hi ha risc d'excedir el límit de velocitat
-    cs = Upozorňovat pokud hrozí riziko překročení rychlosti
-    da = Advar hvis der er en risiko for, at hastighedsgrænsen overskrides
-    de = Warnen wenn die Gefahr einer Geschwindigkeitsüberschreitung besteht
-    el = Να ειδοποιούμαι αν υπάρχει κίνδυνος να υπερβώ το όριο ταχύτητας
-    es = Alertar si existe el riesgo de exceder el límite de velocidad
-    es-MX = Avisar si existe el riesgo de exceder el límite de velocidad
+    comment = "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit
+    tags = android,ios
+    en = Warn if exceeding the speed limit
+    af = Waarsku teen spoed
+    ar = التحذير من السرعة
+    az = Sürətlə bağlı xəbərdarlıq edin
+    be = Папярэджваць калі хуткасць перавышана
+    bg = Да ви предупреди за превишена скорост
+    ca = Avisa’m en excedir velocitat límit
+    cs = Varování před překročením rychlosti
+    da = At advare dig om hastighedsoverskridelser
+    de = Um dich vor Geschwindigkeitsüberschreitungen zu warnen
+    el = Για να σας προειδοποιήσει για υπερβολική ταχύτητα
+    es = Alertar al exceder velocidad límite
+    es-MX = Avisar al exceder velocidad límite
```

Comment:
https://github.com/organicmaps/organicmaps/pull/7585#issuecomment-1996627719 @fitojb suggests to remove `es-MX`:

```suggestion
    es = Alertar al exceder velocidad límite
```

Can you choose between "Alertar al exceder velocidad límite" and "Alertar al exceder límite de velocidad" @fitojb?

Reply:
@fitojb: In `es` we're using the verb "Alertar":

![ios_speed_cam_prefs_es4](https://github.com/organicmaps/organicmaps/assets/25888296/3731b7a1-af0c-411e-b4b3-3cc975053f4f)

While in `es-MX` we're using the verb "Avisar":

![ios_speed_cam_prefs_es-MX1](https://github.com/organicmaps/organicmaps/assets/25888296/265b8f17-7ee3-4e72-93ba-2460a010b16d)

My plan is to keep the wording as shown in these screenshots.